atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/CassandraMultinodeEteTest.java

```diff
@@ -0,0 +1,228 @@
+/**
```

Comment:
Actually, don't do that. I got confused because "CassandraMultinode" means both of:

* "multiple Atlas nodes using the Cassandra KVS" (in CassandraMultinodeTestSuite)
* "multiple Cassandra nodes and one ETE node" (in this file, CassandraMultinodeEteTest)

Instead, we probably want to create a new TestSuite that represents this setup, and that contains these extra tests.